### ISSUE TITLE
Disable Top Streaming Bar in Windows 10 XBox App

### ISSUE THREAD

**Issue by redline99cs** [fqlx/XboxKeyboardMouse#103]
Hello  i would like the app to display fullscreen without the Top Bar Info "Streaming from...."
As soon as i move the mouse the bar appears.
Can this be disabled please?
Also what is the recommended DPI setting for my mouse?
Im having a hard time with 800dpi, 1500DPI semms to be easier to aim.
I run the calibration tool but aiming is still wobly and difficult.
I own a corsair scimitar pro if it helps.
Thank you and great programm

**Comment by DavidRieman** [comment]
We don't own the Xbox streaming app, so we wouldn't be able to disable its behavior in a safe way.  For users with multiple monitors we could probably bind the mouse for measuring over the other screen though.

I don't know the recommended DPI settings, but I did just notice the dead-zoning mode isn't respecting the in-app mouse sensitivity values, and I did just reproduce the wonky/wobbly aiming - so I really hope to be able to debug and tackle that this week.
